src/transformers/utils/quantization_config.py

```diff
@@ -1082,6 +1081,23 @@ def get_loading_attributes(self):
         return loading_attibutes_dict
 
 
+def _get_torchao_quant_type_to_method():
+    if is_torchao_available():
+        from torchao.quantization import (
+            int4_weight_only,
+            int8_dynamic_activation_int8_weight,
+            int8_weight_only,
+        )
+
+        return {
+            "int4_weight_only": int4_weight_only,
+            "int8_weight_only": int8_weight_only,
+            "int8_dynamic_activation_int8_weight": int8_dynamic_activation_int8_weight,
+        }
+    else:
+        raise ValueError("TorchAoConfig requires torchao to be installed, please install with `pip install torchao`")
+
+
```

Comment:
Can you put that in the `TorchAOConfig` class ? 
Otherwise, we can keep that we did before but we just need to make sure to pop `_STR_TO_METHOD` key when converting to a dict. The following should work: 


```py 
    def to_dict(self) -> Dict[str, Any]:
        """
        Serializes this instance to a Python dictionary. Returns:
            `Dict[str, Any]`: Dictionary of all the attributes that make up this configuration instance.
        """
        output = copy.deepcopy(self.__dict__)
        # not json serializable
        output.pop("_STR_TO_METHOD", None)

        return output
```

Reply:
OK will move to `TorchAOConfig` class